Softeq.XToolkit.WhiteLabel.Forms/Converters/EqualityToBooleanConverter.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Developed by Softeq Development Corporation
+// http://www.softeq.com
+
+using System;
+using System.Globalization;
+using Xamarin.Forms;
+
+namespace Softeq.XToolkit.WhiteLabel.Forms.Converters
+{
+    /// <summary>
+    ///     Convert equality between value and parameter to boolean result:
+    ///     returns true if value equals parameter, false otherwise.
+    ///     ConvertBack is not implemented.
+    /// </summary>
+    public class EqualityToBooleanConverter : IValueConverter
```

Comment:
Could you please help with a better understanding of which case it can be used?

Reply:
Added comment